docs/docs-site/content/user/concept/_index.md

```diff
@@ -110,15 +110,96 @@ Clicking on the link will open-up the SQL selection:
 
 ![Click Gist](https://cdn.gethue.com/uploads/2020/03/editor_gist_open_presentation_mode.png)
 
-### Share to Slack
+### Import / Export
+
+Via the Home page, saved documents can be exported for backups or transferring to another Hue.
+
+## Slack
 Currently in **Beta**
 
-This integration increases the collaboration with others via Slack. It expands the ability to share query links or gists to the desired Slack channels which then unfurls in a rich preview for other members to view.
+This integration increases the collaboration with other users via Slack.
 
+### Architectural View
```

Comment:
This is good, but move to Admin section and not User guide? (the later is purely only about using the app)

Reply:
Can you suggest a better location in the Admin section to move the Architectural View there @romainr ?
Do I need to create a separate Slack section in the Admin section too?

Reply:
Moving the `Architecture View` along with the `Manual Slack App Installation` in a separate `Slack` section @romainr.
This will make User guide having completely using the app info and Admin section having the architecture and how to manually follow installation steps